samples/excel/89-preview-apis/chart-axis.yaml

```diff
@@ -0,0 +1,246 @@
+            valueAxis.title.text = \"Number of items\";
+            valueAxis.hasTitle = true;
+            valueAxis.minorGridlines.visible = true;
+    
+            categoryAxis.load();
+            categoryAxis.title.load();
+            valueAxis.load();
+            valueAxis.minorGridlines.load();
+    
+            await context.sync();
+    
+            console.log(\"The category axis title is: \" + categoryAxis.title.text);
+            console.log(\"The value axis title is: \" + valueAxis.title.text);
+            
+            OfficeHelpers.UI.notify(\"The minor gridlines visibility is set to: \" + valueAxis.minorGridlines.visible);
+        });
+    }
+    
+    async function setup() {
+        await Excel.run(async (context) => {
+            const sheet = await OfficeHelpers.ExcelUtilities
+                .forceCreateSheet(context.workbook, \"Sample\");
+            let expensesTable = sheet.tables.add('A1:E1', true);
+            expensesTable.name = \"SalesTable\";
+    
+            expensesTable.getHeaderRowRange().values = [[\"Product\", \"Qtr1\", \"Qtr2\", \"Qtr3\", \"Qtr4\"]];
+    
+            expensesTable.rows.add(null, [
+                [\"Frames\", 5000, 7000, 6544, 4377],
+                [\"Saddles\", 400, 323, 276, 651],
+                [\"Brake levers\", 12000, 8766, 8456, 9812],
+                [\"Chains\", 1550, 1088, 692, 853],
+                [\"Mirrors\", 225, 600, 923, 544],
+                [\"Spokes\", 6005, 7634, 4589, 8765]
+            ]);
+    
+            if (Office.context.requirements.isSetSupported(\"ExcelApi\", 1.7)) {
+                sheet.getUsedRange().format.autofitColumns();
+                sheet.getUsedRange().format.autofitRows();
+            }
+    
+            createChart(context);
+            addVerticalAxisLabel(context);
+            sheet.activate();
+    
+            await context.sync();
+        });
+    }
+    
+    async function createChart(context: Excel.RequestContext) {
+        const sheet = context.workbook.worksheets.getItem(\"Sample\");
+        const salesTable = sheet.tables.getItem(\"SalesTable\");
+    
+        const dataRange = salesTable.getDataBodyRange();
+    
+        let chart = sheet.charts.add(\"ColumnClustered\", dataRange, Excel.ChartSeriesBy.columns);
+    
+        chart.setPosition(\"A15\", \"F30\");
+        chart.title.text = \"Quarterly sales chart\";
+        chart.legend.position = \"right\"
+        chart.legend.format.fill.setSolidColor(\"white\");
+        chart.dataLabels.format.font.size = 15;
+        chart.dataLabels.format.font.color = \"black\";
+        let points = chart.series.getItemAt(0).points;
+        points.getItemAt(0).format.fill.setSolidColor(\"pink\");
+        points.getItemAt(1).format.fill.setSolidColor(\"indigo\");
+    }
+    
+    async function addVerticalAxisLabel(context: Excel.RequestContext) {
+    
+            const sheet = context.workbook.worksheets.getItem(\"Sample\");
```

Comment:
Something is wrong with the tabbing here

Reply:
Looks like the global search/replace we did to try to fix rendering corrupted that yaml file.